kbase-extension/static/kbase/js/widgets/appWidgets2/validators/workspaceObjectName.js

```diff
@@ -58,7 +58,7 @@ define(['bluebird', 'kb_service/utils', 'kb_service/client/workspace'], (
                 messageId = 'obj-name-too-long';
                 diagnosis = 'invalid';
                 errorMessage = 'an object name may not exceed 255 characters in length';
-            } else if (constraints.shouldNotExist) {
+            } else if (constraints.shouldNotExist || options.shouldNotExist) {
```

Comment:
The other part of the change - modifying the `constraints` object of the spec seemed like a bad idea in the evaluation function, but adding another options field is innocuous.